RadarSDK/Radar.h

```diff
@@ -118,260 +118,262 @@ typedef NS_ENUM(NSInteger, RadarRouteUnits) {
 
 /**
- Called when a geofence search request succeeds, fails, or times out. Receives the request status and, if successful, the location and an array of geofences sorted by distance.
+ Called when a geofence search request succeeds, fails, or times out. Receives the request status and, if successful, the location and an array of geofences
+ sorted by distance.
 
  @see https://radar.io/documentation/api#search-geofences
  */
-typedef void(^ _Nonnull RadarSearchGeofencesCompletionHandler)(RadarStatus status, CLLocation * _Nullable location, NSArray<RadarGeofence *> * _Nullable geofences);
+typedef void (^_Nonnull RadarSearchGeofencesCompletionHandler)(RadarStatus status,
+                                                               CLLocation *_Nullable location,
+                                                               NSArray<RadarGeofence *> *_Nullable geofences);
 
 /**
  Called when a geocoding request succeeds, fails, or times out. Receives the request status and, if successful, the geocoding results (an array of addresses).
 
  @see https://radar.io/documentation/api#geocode
  */
-typedef void(^ _Nonnull RadarGeocodeCompletionHandler)(RadarStatus status, NSArray<RadarAddress *> * _Nullable addresses);
+typedef void (^_Nonnull RadarGeocodeCompletionHandler)(RadarStatus status, NSArray<RadarAddress *> *_Nullable addresses);
 
 /**
  Called when an IP geocoding request succeeds, fails, or times out. Receives the request status and, if successful, the geocoding result (a country).
 
  @see https://radar.io/documentation/api#geocode-ip
  */
-typedef void(^ _Nonnull RadarIPGeocodeCompletionHandler)(RadarStatus status, RadarRegion * _Nullable country);
+typedef void (^_Nonnull RadarIPGeocodeCompletionHandler)(RadarStatus status, RadarRegion *_Nullable country);
 
 /**
  Called when a routing request succeeds, fails, or times out. Receives the request status and, if successful, the routes.
 
  @see https://radar.io/documentation/api#route
  */
-typedef void(^ _Nonnull RadarRouteCompletionHandler)(RadarStatus status, RadarRoutes * _Nullable routes);
+typedef void (^_Nonnull RadarRouteCompletionHandler)(RadarStatus status, RadarRoutes *_Nullable routes);
 
 /**
  The main class used to interact with the Radar SDK.
- 
+
  @see https://radar.io/documentation/sdk
  */
 @interface Radar : NSObject
 
 /**
  Initializes the Radar SDK.
- 
+
  @warning Call this method from the main thread in your `AppDelegate` class before calling any other Radar methods.
- 
+
  @param publishableKey Your publishable API key.
- 
+
  @see https://radar.io/documentation/sdk#ios-initialize
  */
-+ (void)initializeWithPublishableKey:(NSString * _Nonnull)publishableKey
-    NS_SWIFT_NAME(initialize(publishableKey:));
++ (void)initializeWithPublishableKey:(NSString *_Nonnull)publishableKey NS_SWIFT_NAME(initialize(publishableKey:));
```

Comment:
what're your thoughts on having `NS_SWIFT_NAME` on the same line as the last param? I kinda prefer it on a new line - is there a setting we can enable to do that?

Reply:
hmm I don't see clang-format has any style options for that specifically https://clang.llvm.org/docs/ClangFormatStyleOptions.html

Reply:
yeah, no worries, I've seen differing standards, even on SO / iOS Documentation, should be fine as is - https://developer.apple.com/documentation/swift/objective-c_and_c_code_customization/renaming_objective-c_apis_for_swift